### PR TITLE
processing data in models

### PR DESCRIPTION
**Pull request by AlierPM**
# Created models
- [x] Create model classes for all entities as shown in the [ERD diagram](https://github.com/microverseinc/curriculum--rails/blob/main/blog-app/images/blog_app_erd.png).
- [x] Set up associations between models.
- [x] Used models to insert data
- [x] Created custom methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,8 +180,10 @@ To run tests, run the following command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- Features -->
-### Key Features <a name="key-features"></a>
```
Comment (KDlamini) [inline README.md:184]: Well done documenting the README. Everything looks great 👏🏾👍🏾

- Please add the `Key Features`  section you deleted and make sure that you have at least [1 - 3] features listed. Please never remove any headings and sub-headings on the template except for the FAQ section. It is the only one that is optional.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @AlierPM 🙋‍♀️,  

Great work so far! 💯 
You have done a great job processing data in models for the blog app. 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Professional Pull Request
✔️  Model associations set up correctly
✔️  All required model methods implemented
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AlierPM) [reply README.md:184]: Thanks @KDlamini ; resolution considered

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @AlierPM, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**